confirm-password.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Confirm Password</title>
+		<link rel="stylesheet" href="styles.css">
+		<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+	</head>
+	<body>
+		<section>
+			<h2>Enter Password</h2>
+			<div class="password-container">
+				<input type="password" name="password" id="password" class="">
```

Comment:
Think that's because I used index.html as the skeleton for confirm-password.html. I believe the empty class (class="")  can be found in index.html as well. Thanks for the heads-up, will remove the empty classes in confirm-password.html in the next commit!